etc/config/cuda.amazon.properties

```diff
@@ -39,9 +39,21 @@ compiler.nvcc11.semver=11.0.2
 compiler.nvcc11u1.exe=/opt/compiler-explorer/cuda/11.0.3/bin/nvcc
 compiler.nvcc11u1.options=--compiler-bindir /opt/compiler-explorer/gcc-9.3.0/bin
 compiler.nvcc11u1.semver=11.0.3
-compiler.nvc111.exe=/opt/compiler-explorer/cuda/11.1.0/bin/nvcc
-compiler.nvc111.options=--compiler-bindir /opt/compiler-explorer/gcc-10.2.0/bin
-compiler.nvc111.semver=11.1.0
+compiler.nvcc111.exe=/opt/compiler-explorer/cuda/11.1.0/bin/nvcc
+compiler.nvcc111.options=--compiler-bindir /opt/compiler-explorer/gcc-10.2.0/bin
+compiler.nvcc111.semver=11.1.0
```

Comment:
this will need an extra `compiler.nvcc111.alias=nvc111` in case it was ever used in a shortlink

Reply:
I don't quite understand. What's different about this one that it needs the alias? Do the other ones need it too?

Reply:
this one was renamed from nvc111 to nvcc111

Reply:
other ones do not need it, just existing id's that are changed